src/Compositor.hpp

```diff
@@ -170,7 +167,7 @@ class CCompositor {
 
     uint64_t         m_iHyprlandPID    = 0;
     wl_event_source* m_critSigSource   = nullptr;
-    rlimit           m_sOriginalNofile = {0};
+    rlimit           m_sOriginalNofile = {.rlim_cur = 0};
```

Comment:
not the same!!!!!!

Reply:
hmmmm?

first elem is rlim_cur on my machine. Or it is intentional UB here?
```
struct rlimit
  {
    /* The current (soft) limit.  */
    rlim_t rlim_cur;
    /* The hard limit.  */
    rlim_t rlim_max;
  };
```

Reply:
I just want to figure it out before making edits.

Reply:
woops, my bad. read up and it apparently is the same, but you can also just do `{}` and it will get zeroes.

Reply:
it probably can be written as
```
    rlimit           m_sOriginalNofile;
```
in this case

Reply:
according to https://en.cppreference.com/w/cpp/language/zero_initialization

Reply:
no, that will leave it uninitialized.

Reply:
can you just make it {}

Reply:
oh sorry, just forgot to push